tests/com/haskforce/highlighting/annotation/external/GhcUtilTest.java

```diff
@@ -0,0 +1,53 @@
+package com.haskforce.highlighting.annotation.external;
+
+import com.intellij.openapi.editor.LogicalPosition;
+import com.intellij.openapi.editor.VisualPosition;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by kasper on 12/24/14.
```

Comment:
I love JavaDoc. Can you drop your name/date (we have that in Git annotate) and add documentation that is more in line with the JavaDoc in other similar files?